libqtile/widget/backlight.py

```diff
@@ -108,24 +113,22 @@ def _load_file(self, path):
 
     def _get_info(self):
         brightness = self._load_file(self.brightness_file)
-
-        info = {
-            'brightness': brightness,
-            'max': self.max_value,
-        }
-        return info
+        max_value = self._load_file(self.max_brightness_file)
+        return 100 * brightness / max_value
```

Comment:
Should we keep the dict format here?

Reply:
What for? _get_info is only used when we want the percentage

Reply:
Ah sorry, I thought it was related to the IPC.